code/modules/client/client_procs.dm

```diff
@@ -797,6 +797,9 @@ GLOBAL_LIST_EMPTY(external_rsc_urls)
 		// so that the visual focus indicator matches reality.
 		winset(src, null, "input.background-color=[COLOR_INPUT_DISABLED]")
 
+	if(!prefs.hotkeys)
```

Comment:
Are you sure this is right?

Reply:
Yes, when hotkey mode is enabled prefs.hotkey is 1

Reply:
```suggestion
	else
```